django_test_suite.sh

```diff
@@ -4,20 +4,45 @@
 # Use of this source code is governed by a BSD-style
 # license that can be found in the LICENSE file.
 
-# exit when any command fails
-set -e
+set -x pipefail
+
+sudo apt-get update -y
+sudo apt-get install -y libmemcached-dev
+
+# Disable buffering, so that the logs stream through.
+export PYTHONUNBUFFERED=1
+
+if [ $SPANNER_EMULATOR_HOST != 0 ]
+then
+    pip3 install .
+    pip3 install -e 'git+https://github.com/q-logic/python-spanner.git@django_test_fixes#egg=google-cloud-spanner'
+    pip3 install -e 'git+https://github.com/q-logic/python-spanner-django.git@dj_tests_against_emulator#egg=django-google-spanner'
+fi
+
+export DJANGO_TESTS_DIR="django_tests_dir"
+mkdir -p $DJANGO_TESTS_DIR && git clone --depth 1 --single-branch --branch spanner-2.2.x https://github.com/timgraham/django.git $DJANGO_TESTS_DIR/django
```

Comment:
Another reminder, we have to fix this in https://github.com/googleapis/python-spanner-django/issues/485.